lib/csharp/src/Protocol/TCompactProtocol.cs

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License. You may obtain a copy of the License at
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why are we doing that change?

Reply:
This was by accident. I fixed it. thank you!